### ISSUE TITLE
Add schematic for transloco-keys-manager does not observe configured package manager.

### ISSUE THREAD

**Issue by napei** [ngneat/transloco#317]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ x ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

This is an issue with the schematic for [ngneat/transloco-keys-manager](https://github.com/ngneat/transloco-keys-manager), however the schematic is in this repo so I'll add the issue here.

The schematic explicitly uses npm to install packages, even though it's explicitly set in `angular.json` that I use `yarn`, and a `yarn.lock` is present. This is kind of annoying as I need to then remove the `package-lock` file and then reinstall with `yarn` again.

[https://github.com/ngneat/transloco/blob/c5281191bab02b516df9d4cc2ab4711d25836678/schematics/src/keys-manager/index.ts#L11](https://github.com/ngneat/transloco/blob/c5281191bab02b516df9d4cc2ab4711d25836678/schematics/src/keys-manager/index.ts#L11)

## Expected behavior

I expect the `ng add` schematic to use whatever angular is configured to use which I believe is available under `NodePackageInstallTask()` in `@angular-devkit/schematics/tasks`

## Minimal reproduction of the problem with instructions

- Generate new angular application

- Set package manager to yarn with `ng config cli.packageManager yarn` or manually by editing `angular.json` and adding the following:

 ```json
  "cli": {
     "packageManager": "yarn"
   }
```

- Install `transloco-keys-manager` using `ng add @ngneat/transloco-keys-manager`

## What is the motivation / use case for changing the behavior?

Annoying that it doesn't follow the angular configuration.

## Environment

<pre><code>
Angular version: 10.0.5

Browser:
N/A

For Tooling issues:
- Node version: 14.6.0  
- Platform: Windows  

Others:

</code></pre>

**Comment by itayod** [comment]
@napei would you like to create a PR for that?

**Comment by napei** [comment]
@itayod Just gave it a go but couldn't get anything to build properly on my machine. Also I'm on windows and `cp` doesn't exist so the copy npm scripts don't work. Was also getting a `TypeError: this.typeChecker.getTypeArguments is not a function` from `tsickle`. It's probably more efficient for you guys to add it. Sorry about that 😢 

Basically I got as far as adding this in `index.ts` for `keys-manager`

```ts

import { addPackageJsonDependency, NodeDependency, NodeDependencyType } from '@schematics/angular/utility/dependencies';
import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';

function getKeysManagerDependency(): NodeDependency {
  const { VERSION } = require('@ngneat/transloco-keys-manager');
  const dep: NodeDependency = {
    name: '@ngneat/transloco-keys-manager',
    version: `^${VERSION.major}`,
    overwrite: false,
    type: NodeDependencyType.Dev
  };
  return dep;
}

function getNgxBuildPlusDependency(): NodeDependency {
  const { VERSION } = require('ngx-build-plus');
  const dep: NodeDependency = {
    name: 'ngx-build-plus',
    version: `^${VERSION.major}`,
    overwrite: false,
    type: NodeDependencyType.Dev
  };
  return dep;
}

function installKeysManager(tree: Tree, context: SchematicContext) {
  addPackageJsonDependency(tree, getKeysManagerDependency());
  addPackageJsonDependency(tree, getNgxBuildPlusDependency());
  context.addTask(new NodePackageInstallTask());
  context.logger.log('info', `🔍 Installing packages...`);
}
```

Haven't been able to test this, however I believe this may be quickest approach.

IMO the add schematic should probably live inside `@ngneat/transloco-keys-manager` anyway if possible as it makes more sense to just call `ng add @ngneat/transloco-keys-manager` rather than `ng add @ngneat/transloco:`

**Comment by itayod** [comment]
@napei you can try running this script using git bash, it should work even if you work in windows :)

**Comment by napei** [comment]
@itayod PR made in #322

**Comment by itayod** [comment]
fixed in v2.19.2
